internal/pkg/remote/manager.go

```diff
@@ -145,15 +145,6 @@ func (u *UnitOfWork) create(objectState model.ObjectState, object model.Object)
 		OnSuccess(func(response *client.Response) {
 			// Save new ID to manifest
 			objectState.SetRemoteState(object)
-
-			// If marking as deleted -> local state is not set
```

Comment:
Pri predchadzajucich taskoch, sa zabudlo toto vyhodit.
Config/row sa uz neoznacuju v DEV branch s `[TO DELETE]`, ale uz sa daju priamo mazat.
Takze tento kod tu uz byt nema.